server/serializers/subscription_filters.go

```diff
@@ -0,0 +1,19 @@
+package serializers
+
+type GitRepositoriesResponse struct {
+	Value []GitRepositories `json:"value"`
+}
+
+type GitRepositories struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type GitBranchesResponse struct {
+	Value []GitBranches `json:"value"`
+}
+
+type GitBranches struct {
+	ID   string `json:"objectId"`
+	Name string `json:"name"`
+}
```

Comment:
```suggestion
type GitBranchesResponse struct {
	Value []GitBranch `json:"value"`
}

type GitBranch struct {
	ID   string `json:"objectId"`
	Name string `json:"name"`
}
```
I think this would be a better name?